guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -103,7 +108,12 @@ Without the option, AD users are unable to use `kinit` to authenticate to {Proje
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
+ifdef::foremanctl[]
+# foremanctl deploy --external-authentication=ipa
```

Comment:
Style thing that we should decide on: `--pam-service foreman-prod` is used earlier and here `--external-authentication=ipa`. I'm leaning to leaving out `=` which we also do in our current installer documentation.